js_modules/dagit/packages/ui/src/components/Suggest.tsx

```diff
@@ -55,7 +55,7 @@ export const Suggest = <T,>(props: React.PropsWithChildren<SuggestProps<T>>) =>
     ...props.popoverProps,
     minimal: true,
     modifiers: deepmerge(
-      {offset: {enabled: true, offset: '0, 8px'}},
```

Comment:
I baked this into our core component because we only have two Suggests and both of them were passing this override so the dropdown aligns with the bottom of the input better.